recipes/rotor/all/conanfile.py

```diff
@@ -67,52 +77,39 @@ def validate(self):
         compiler = str(self.settings.compiler)
         if compiler not in minimal_version:
             self.output.warn(
-                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
+                f"{self.name} recipe lacks information about the {compiler} compiler standard version support")
```

Comment:
Better put put for 2.0

```suggestion
                f"{self.ref} recipe lacks information about the {compiler} compiler standard version support")
```